### ISSUE TITLE
Driver Problems

### ISSUE THREAD

**Issue by PMTGaming** [pico-coder/sigrok-pico#50]
Hello,

I have transferred the pico_sdk_sigrok.uf2 file onto the Raspberry Pico and installed the PulseView version from the Dropbox link provided in the README. I used Zadig to install the USB serial CDC driver, but I am encountering the following error: "The device cannot start. (Code 10)."

Additionally, it states: "A device which does not exist was specified."

![grafik](https://github.com/pico-coder/sigrok-pico/assets/13299095/74f599c4-02fe-4359-b5ab-96dc38359160)

![Screenshot 2024-06-27 161250](https://github.com/pico-coder/sigrok-pico/assets/13299095/3208befc-ac42-4454-8e22-97cf1f685cea)

**Comment by pico-coder** [comment]
I updated Readme.md to mention that the dropbox link and my windows builds instructions should be avoided, and the mainline release at sigrok.org/wiki/Downloads should be used.
I also added some possible things to try in https://github.com/pico-coder/sigrok-pico/blob/main/GettingStarted.md when the device doesn't work on windows.
However, most of the serial stuff is out of my control, as it is part of windows, or the libserial library etc.  Further, I have no way of debugging serial device failures under windows.  Usually plugging/unplugging/rebooting eventually works.
